umbra-js/test/KeyPair.test.ts

```diff
@@ -76,9 +76,9 @@ describe('KeyPair class', () => {
   describe('Functionality', () => {
     it('will recover the public key from an arbitrary transaction', async () => {
       // Specify rinkeby transaction hash and its sender
```

Comment:
maybe worth searching all test files for "rinkeby" to make sure they're all fixed
```suggestion
      // Specify sepolia transaction hash and its sender
```

Reply:
Cleaned up rest of the rinkeby stuff here: https://github.com/ScopeLift/umbra-protocol/pull/521/commits/deefd4ff6a535b7855e1c8735991cc61b08ba28f